src/metaxy/graph/diff/models.py

```diff
@@ -13,6 +13,11 @@
     from metaxy.models.feature import FeatureGraph
 
 
+# Sentinel value for None code_version in struct serialization
+# (Polars structs require a non-None value, so we use "0" to represent missing/None)
+EMPTY_CODE_VERSION = "0"
```

Comment:
Rename to DEFAULT_CODE_VERSION (it's not empty!) and let's set it to "__metaxy__intitial__"